packages/@o3r/pipeline/testing/jest.config.ut.builders.js

```diff
@@ -0,0 +1,16 @@
+const path = require('node:path');
+const getJestProjectConfig = require('../../../../jest.config.ut').getJestProjectConfig;
+const rootDir = path.join(__dirname, '..');
+
+/** @type {import('ts-jest/dist/types').JestConfigWithTsJest} */
+module.exports = {
+  ...getJestProjectConfig(rootDir, false),
+  displayName: require('../package.json').name,
+  rootDir,
+  testPathIgnorePatterns: [
+    '<rootDir>/.*/templates/.*',
+    '<rootDir>/src/.*',
```

Comment:
why do you ignore this folder? (env if there is currently no test in it)

Reply:
this is the `ut.builders.js` file. No Builder tests should be under `/src` right?